README.md

```diff
@@ -25,7 +25,6 @@ Clone the [source code repository](https://github.com/Sage-Bionetworks/annotator
 
     git clone https://github.com/Sage-Bionetworks/annotator.git
     cd annotator
-    git checkout develop
```

Comment:
It looks like these instructions are specifically if you _want_ to use the develop branch, so I think this line needs to stay.

Reply:
I agree, but let's spell that out directly - if we want all development work to happen on 'develop' then we can state it explicitly and require it as part of a contribution guidelines. We could also have people fork annotator, handle their dev work the way they want, and PR in master (similar to the synapseAnnotations contribution workflow). I guess that goes back to @philerooski 's point: do we want annotator maintained as a project with specific up-to-date goal(s) or rather have it as an ad-hoc package of hodgepodge utilities used internally from time to time (with required bug fixes each of the 2 times a year it's used).  In the latter case I'd be ok with not having contribute guidelines and be somewhat lax w/ overall code requirements, as long as whatever is in master does its job - e.g. updates synapse annotation table when we do a new release. If we want to actively maintain the package - changing/adding/removing features; keeping up-to-date with dependencies, etc. - then having a bit more formal contribution requirements would be good?

Reply:
Agree, but we shouldn't make the instructions inaccurate in the meantime. This section is about how to install the develop version, which requires `git checkout develop`. 

> We could also have people fork annotator, handle their dev work the way they want, and PR in master (similar to the synapseAnnotations contribution workflow).

I am fine with this, and in this case the whole "install develop branch" section of the readme could be deleted, but just a note that this _isn't_ the synapseAnnotations workflow -- people submit PRs to develop there.

Reply:
Sounds good. Do we want to try discussing who's using/maintaining annotator; and what are the main use cases we want it to handle in the future - @philerooski @kdaily?